cmd/collector/app/span_processor_test.go

```diff
@@ -383,6 +383,142 @@ func TestSpanProcessorWithCollectorTags(t *testing.T) {
 	assert.Equal(t, expected.Process, span.Process)
 }
 
+func TestSpanProcessorWithCollectorTagsWithUseOnTagKeyConflictAsBoth(t *testing.T) {
+	testCollectorTags := map[string]string{
+		"extra": "tag",
+		"env":   "prod",
+		"node":  "172.22.18.161",
+	}
+
+	w := &fakeSpanWriter{}
+	p := NewSpanProcessor(w, nil, Options.CollectorTags(testCollectorTags), Options.UseOnTagKeyConflict("both")).(*spanProcessor)
```

Comment:
Given `"both"` is the default, I suggest adding (maybe with a table driven test), a test where the `UseOnTagKeyConflict` option isn't provided, which we should expect to have the same result as `"both"` being explicitly set.

Reply:
There is a test in the master for that: https://github.com/jaegertracing/jaeger/blob/master/cmd/collector/app/span_processor_test.go#L338